framework/src/media/MediaRecorderImpl.h

```diff
@@ -126,7 +128,9 @@ class MediaRecorderImpl : public enable_shared_from_this<MediaRecorderImpl>
 	unsigned char* mBuffer;
 	int mBuffSize;
 	mutex mCmdMtx; // command mutex
+	mutex mObMtx; // observer mutex
 	std::condition_variable mSyncCv;
+	std::condition_variable mObFlushCv;
```

Comment:
instead of flush, how about below?
mCmdSyncCv, mObSyncCv;
also, please consider rename notifySync

Reply:
thanks @Taejun-Kwon ,
i will close this commit.